syntax.go

```diff
@@ -89,6 +93,14 @@ func renderPaste(w http.ResponseWriter, text, title, author, notes []byte) {
 		}
 
 		if len(m[6]) != 0 {
+			if p, ok := pokemonData[string(m[6])]; ok {
```

Comment:
Not keen on just defaulting to male sprites if gender's unspecified - we can be a bit more creative than that.